openpaas-james/mailbox/opendistro/src/test/java/org/apache/james/mailbox/opendistro/OpenDistroClusterExtension.java

```diff
@@ -0,0 +1,118 @@
+/****************************************************************
```

Comment:
IMO we do not need to duplicate the Cluster tests : it takes time, we already have such tests running on v7.

Reply:
It is being used in the only test with authentication. If we remove it, i will have to make changes to other tests in order to check if we can authenticate with OpenDistro - which is not the point of this ticket i think. Another PR may be?